include/edm4hep/GenToolInfo.h

```diff
@@ -0,0 +1,50 @@
+#ifndef EDM4HEP_GENTOOLINFO_H
+#define EDM4HEP_GENTOOLINFO_H
+
+#include "edm4hep/Constants.h"
+#include "podio/Frame.h"
+#include <string>
+#include <vector>
+
+namespace edm4hep {
+
+struct GenToolInfo {
```

Comment:
The classes we define in the yaml file are all prefixed `Generator`, but this one is `Gen`. For consistency reasons, I think they should both be the same. The choice is also related with how we want to name the constants.